projects/npm-tools/packages/npm-scripts/src/utils/runWebpackAsBundler.js

```diff
@@ -146,7 +146,7 @@ function getIndexWebpackConfig(projectDir, buildConfig, babelConfig) {
 	};
 
 	if (process.env.NODE_ENV === 'development') {
-		webpackConfig.devtool = 'cheap-source-map';
+		webpackConfig.devtool = 'source-map';
```

Comment:
This seems pretty expensive to me...

![](https://media.giphy.com/media/q8pghZNgAQ0rm/giphy.gif)

Reply:
![clap](https://media.giphy.com/media/nbvFVPiEiJH6JOGIok/giphy.gif)